package.json

```diff
@@ -24,6 +24,7 @@
     "opentok-react": "^0.9.0",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
+    "react-loader-spinner": "^3.1.2",
```

Comment:
Added spinner for race condition rendering problems. auth.currentUser was not loading fast enough for the redirect on the mentor_dashboard after loading. If refreshing the page it was redirecting back to the landing. Refer below